packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts

```diff
@@ -489,7 +487,7 @@ export class V8 {
 
     blockSocket.on('connect', () => {
       log.info(`Connected to block ${this.getConnectionInfo()}`);
-      blockSocket.emit('room', `/${this.coin.toUpperCase()}/${this.v8network}/inv`);
+      blockSocket.emit('room', `/${this.chain}/${this.v8network}/inv`);
```

Comment:
this.chain is lowercased up above right? I don't think that'd be a valid room for bitcore-node